### PR TITLE
feat(dialog): add the ability to align the content of md-dialog-actions

### LINKED ISSUE #2483: Feature request: add align attribute to md-dialog-actions
md-card-actions has 'align' attribute that allows us to sort elements. I thought md-dialog-actions has same attribute, but not. It is not necessary, but needs to be implemented for consistency.

### PR DESCRIPTION
**Pull request by crisbeto**
Adds the `align` attribute, which can be set to `end` or `middle`, that allows users to align the content of the `md-dialog-actions`.

Fixes #2483.

### CODE REVIEW

File: src/lib/dialog/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ A reference to the dialog created by the MdDialog `open` method.
 | `md-dialog-title`   | Marks the title of the dialog.
 | `md-dialog-content` | Scrollable content of the dialog.
 | `md-dialog-close`   | When added to a `button`, makes the element act as a close button for the dialog.
-| `md-dialog-actions` | Wrapper for the set of actions at the bottom of a dialog. Typically contains buttons.
+| `md-dialog-actions` | Wrapper for the set of actions at the bottom of a dialog. Typically contains buttons. The element's content can be aligned via the `align` attribute either to the `middle` or the `end`. |
```
Comment (jelbourn) [inline src/lib/dialog/README.md]: Should be `center` rather than `middle`.

File: src/lib/dialog/dialog.scss [hunk 2]
```diff
@@ -43,12 +43,20 @@ md-dialog-content, [md-dialog-content], mat-dialog-content, [mat-dialog-content]
...
+  display: flex;
 
   &:last-child {
     // If the actions are the last element in a dialog, we need to pull them down
     // over the dialog padding, in order to avoid the action's padding stacking
     // with the dialog's.
     margin-bottom: -$md-dialog-padding;
   }
+
+  &[align='end'] {
```
Comment (jelbourn) [inline src/lib/dialog/dialog.scss]: If someone sets `align` with a binding (`[align]`="buttonAlignment")` does it still apply this attribute?

Comment (crisbeto) [inline src/lib/dialog/dialog.scss]: It does. That's what I did in the playground/demo app.

File: src/lib/dialog/README.md (hunk 1 above)
Comment (crisbeto) [inline src/lib/dialog/README.md]: Renamed.

**Pull request merged**